webapp/src/codecard.tsx

```diff
@@ -95,9 +102,18 @@ export class CodeCardView extends data.Component<CodeCardProps, CodeCardState> {
         const ariaLabel = card.ariaLabel || card.title || card.shortName || name;
 
         const style = card.style || "card"
-        const cardDiv = <div className={`ui ${style} ${color} ${card.onClick ? "link" : ''} ${className ? className : ''}`}
-            role={card.role} aria-selected={card.role === "option" ? "true" : undefined} aria-label={ariaLabel} title={card.title}
-            onClick={clickHandler} tabIndex={card.onClick ? card.tabIndex || 0 : null} onKeyDown={card.onClick ? fireClickOnEnter : null}>
```

Comment:
`fireClickOnEnter` does the wrong thing here when using "Enter" or "Space" while the "Feedback" anchor tag is focussed. It prevents the default behaviour, then triggers click on this element instead of the anchor tag from which the original event was triggered, causing the experiment to toggle on/off.